assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -5959,7 +5959,9 @@ private void post_save_submission(RunData data, boolean post) {
                         submission.setHonorPledge(Boolean.valueOf(honorPledgeYes));
                         submission.setDateSubmitted(Instant.now());
                         submission.setSubmitted(post);
-                        submission.setUserSubmission(true);
```

Comment:
I think we could remove userSubmission field and just use the same logic that if a submittee is set then its considered a user submission.